assignment.rb

```diff
@@ -0,0 +1,77 @@
+acct_holders = [
+  {
+  name: "Al Anderson",
+	acct: "1234",
+	pin: "1234",
+	funds: 50
+	},
+  {
+	name: "Bob Barker",
+	acct: "5678",
+	pin: "5678",
+	funds: 8000
+	},
+  {
+	name: "Carl Carson",
+	acct: "9090",
+	pin: "9090",
+	funds: 1200
+	}
+]
+atm_balance = 7000
+withdraw_amount = 0
+
+# Login
+current_acct = ""
+while current_acct == ""
+  puts "Please enter your account number"
+    current_acct = gets.chomp
+end
+current_pin = ""
+while current_pin == ""
+  puts "Please enter your PIN"
+    current_pin = gets.chomp
+end
+
+current_user = ""
+  acct_holders.each do |user|
+    if user[:acct] == current_acct && user[:pin] == current_pin 
+    current_user = user
+    puts "Welcome, #{current_user[:name]}."
+    
+# ATM Options Menu
+  if current_user
```

Comment:
if this is supposed to _not_ be nested within anything (like a block), then make sure it's not indented!